frontend/libs/studio-components/src/components/StudioRecommendedNextAction/StudioRecommendedNextAction.test.tsx

```diff
@@ -0,0 +1,85 @@
+import { render, screen } from '@testing-library/react';
+import { StudioRecommendedNextAction } from './StudioRecommendedNextAction';
+import React from 'react';
+import userEvent from '@testing-library/user-event';
+
+describe('StudioRecommendedNextAction', () => {
+  const user = userEvent.setup();
```

Comment:
It's recommended by Testing Library to define user within each `it` or `test` instead. 😊 Ref: https://testing-library.com/docs/user-event/intro/#writing-tests-with-userevent